config/development.js

```diff
@@ -21,13 +21,15 @@ module.exports = {
   },
 
   userToken: {
-    publicKey: '-----BEGIN PUBLIC KEY-----\n' +
+    publicKey:
```

Comment:
Let's not change this please @Kratika0907

Reply:
This has not been changed